app/src/main/res/values/strings.xml

```diff
@@ -107,4 +107,6 @@
     <string name="details">Details</string>
     <string name="tasks">Tasks</string>
     <string name="add_new_task">Add a new task</string>
+    <string name="confirm_logout">CONFIRM LOGOUT</string>
```

Comment:
```suggestion
    <string name="confirm_logout">Confirm Logout</string>
```
For real, I'm not sure if it sholf be "Confirm logout" or "Confirm Logout"

Reply:
I have made it Confirm Logout for now . Do let me know if you want me to change it :)